msktutil.cpp

```diff
@@ -41,6 +41,11 @@
 
 int g_verbose = 0;
 
+/* Fatal error */
+void error_exit( const char *text) {
+    fprintf(stderr, "error_exit: %s: %s\n", text, strerror(errno));
```

Comment:
Does the user really case that `error_exit` is used here? I think all he needs to see is `ERROR`. Everything else is an implementation detail.